### DISCUSSION TITLE
I just wanna ask how to `export`/`save`  tiptap to `docs`/`pdf`?

### DISCUSSION THREAD

**Discussion by JenuelDev** [ueberdosis/tiptap#1995] [General Questions]
Hi! I am actually using tiptap for editing text on my website. Now I am actually searching for a way to export it as docs or pdf. Has anyone tried it? how did you do it? thanks :)

**Comment by philippkuehn** [comment]
you can try https://github.com/curvenote/prosemirror-docx

**Reply by JenuelDev** [reply]
thanks, Have you tried implementing it?

**Reply by philippkuehn** [reply]
nope. not yet.
